src/components/Rockets/SingleRockets.jsx

```diff
@@ -42,11 +57,51 @@ const SingleRocket = ({ allrocekts }) => {
                   CANCEL DRAGON
                 </button>
               )}
-            </div>
+            </td>
+            </tr>
+
+          
+
+          // <div key={rocket.id} className="rocket-card">
+          //   <div className="rocket-card-header">
+          //     <h3>{rocket.rocket_name}</h3>
+          //     <img className="rocket-image" src={rocket.flickrImages} alt={rocket.rocket_name} />
+          //   </div>
+          //   <div className="rocket-card-body">
+          //     {rocket.reserved ? <span className="rocket-unavailable">RESERVED</span> : ' '}
+          //     <p>{rocket.description}</p>
+          //   </div>
 
-          </div>
+          //   <div>
+          //     {rocket.reserved === false ? (
+          //       <button
+          //         className="btn-reserve-rocket"
+          //         onClick={() => dispatch(reserveRocket(rocket.id))}
+          //         id={rocket.id}
+          //         type="submit"
+          //       >
+          //         RESERVE DRAGON
+          //       </button>
+          //     ) : (
+          //       <button
+          //         className="btn-cancel-reservation"
+          //         onClick={() => dispatch(cancelRocket(rocket.id))}
+          //         id={rocket.id}
+          //         type="submit"
+          //       >
+          //         CANCEL DRAGON
+          //       </button>
+          //     )}
+          //   </div>
+
+          // </div>
```

Comment:
HI there my friend we just have some unnecessary commented code, it is all right I will approve it